### PR TITLE
Concept map for functions in programming languages

### PR DESCRIPTION
**Pull request by HKYang**
This concept map is to illustrate functions in programming languages, including their design and use.

### CODE REVIEW

File: _submissions/round-12/1/2015-02-25-functions-in-programming-languages.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  - Function
+  - Programming Language
+---
+SWC link:
+http://swcarpentry.github.io/python-novice-inflammation/04-func.html
+
+This concept map is intended to illustrate functions in programming languages, including the design and use of functions.
+
+Please review and download at 
+[Functions in Programming Languages](http://imgur.com/gWC7JpI)
```
Comment (gvwilson) [inline _submissions/round-12/1/2015-02-25-functions-in-programming-languages.md]: The link will need to be `http://i.imgur.com/gWC7JpI.jpg` to display the image - using `http://i.imgur.com/gWC7JpI` on its own is just a link to the Imgur page.

**Comment by HKYang** [comment]
Make URL link more friendly.

**Comment by gvwilson** [comment]
HI @HKYang - this is showing 37 files changed.  Can you please:
1. go into your training-course repo on your desktop
2. git pull upstream gh-pages (assuming gh-pages is set to point at github.com/swcarpentry/training-course.git)
3. git push origin gh-pages (to get those changes into your repo on GitHub)
This PR should then automatically update.
Thanks,
@gvwilson

**Comment by gvwilson** [comment]
@HKYang - we can't merge this until you update your repo from swcarpentry/training-course::gh-pages so that all the out-of-date files are up to date.

**Comment by gvwilson** [comment]
@HKYang , this still included over 30 files - it should not have been merged.  I'll start work on a fix.

**Pull request merged**